src/test/java/org/broadinstitute/hellbender/tools/spark/validation/CompareDuplicatesSparkIntegrationTest.java

```diff
@@ -53,16 +53,47 @@ public void identicalBamTest(File firstBam, File secondBam) throws Exception {
     }
 
     @Test(dataProvider = "CompareDifferentDuplicatesProvider", expectedExceptions = UserException.class, groups = "spark")
-    public void differentBamTest(File firstBam, File secondBam) throws Exception {
+    public void differentBamTest(File firstBam, File secondBam, File firstBamDiffs, File secondBamDiffs) throws Exception {
         // These files are not the same and should throw an exception
         ArgumentsBuilder args = new ArgumentsBuilder();
         args.add("--" + StandardArgumentDefinitions.INPUT_LONG_NAME);
         args.add(firstBam.getCanonicalPath());
         args.add("--" + CompareDuplicatesSpark.INPUT_2_SHORT_NAME);
         args.add(secondBam.getCanonicalPath());
-        args.add("--" + CompareDuplicatesSpark.THROW_ON_DIFF_LONG_NAME);
-        args.add("true");
+        if (firstBamDiffs != null && secondBamDiffs != null) {
+            args.add("-O");
+            args.add(createTempFile("differentBamTest1", ".bam"));
+            args.add("-O2");
+            args.add(createTempFile("differentBamTest2", ".bam"));
+        } else {
+            args.add("--" + CompareDuplicatesSpark.THROW_ON_DIFF_LONG_NAME);
+            args.add("true");
+        }
+
 
         this.runCommandLine(args.getArgsArray());
     }
-}
+
+    @Test( groups = "spark")
+    public void testOutputFile() throws Exception {
+        // These files are not the same and should throw an exception
```

Comment:
I'm confused by this comment.  It doesn't  look like you're expecting an exception anywhere in this test.

Should this be validating something about the output files?

Reply:
leftover, this was an old test asserting that two non-matching files do indeed find themselves not matching. I have reverted it to its correct state

Reply:
Shouldn't this comment be removed?